### ISSUE TITLE
Calendar example in API documentation is incorrect

### ISSUE THREAD

**Issue by Elliria** [autokey/autokey#645]
## Classification:
Documentation

## Reproducibility:
Always

## AutoKey version:
Any

## Used GUI:
Any

## Installed via:
Any

## Linux distribution:
Any

## Summary:
The calendar example in AutoKey's API documentation uses incorrect syntax.

## Steps to reproduce:
1. Visit the [Class GtkDialog page in the API documentation](https://autokey.github.io/api/gtkdialog.html).
2. Scroll down to the calendar example at the bottom of the page.
3. Look at the example syntax.
## Expected result:
The syntax the example should be using is:
```
Usage: dialog.calendar(title="Choose a date", format="%Y-%m-%d", date="YYYY-MM-DD", **kwargs)
```
## Actual result:
The syntax shown in the example is:
```
Usage: dialog.calendar_dialog(title="Choose a date", format="%Y-%m-%d", date="YYYY-MM-DD", **kwargs)
```
## Notes:
I'd be happy to help with issues like this if I can get some kind of access to the documentation.

**Comment by sebastiansam55** [comment]
Does using `calendar_dialog` work for you? It doesn't look like it should, the function hasn't been changed in the develop branch or the master branch, it should still be `dialog.calendar`

**Comment by Elliria** [comment]
No, ```calendar_dialog``` doesn't work for me. If the code hasn't changed, then the title and summary of this issue should be edited to it being just a typo.

**Comment by Elliria** [comment]
I updated the title and summary.

**Comment by sebastiansam55** [comment]
Oh, I see what you're saying now, well in any case we'll hopefully get the new documentation figured out at some point soon, it's currently live here https://autokey.github.io/autokey/

**Comment by Elliria** [comment]
Ah, thanks. Well, I clicked the "Edit on GitHub" link they provide and got a 404, so I guess it has to be gotten to from somewhere else.

**Comment by josephj11** [comment]
@Elliria Code Triage just landed me here. I have stared at the two API calls above and can't see any difference between them. I don't have the sample scripts installed on my system, so I couldn't easily check the script itself.

Can we close this or is it still an issue?

**Comment by Elliria** [comment]
This is still an issue and is bigger than I had originally thought. The link in my report is no longer any good since the documentation was updated. That's corrected below.

The problem exists in the **Usage** line in the legacy documentation:
* It's in the GtkDialog API page [here](https://autokey.github.io/autokey/lib.scripting.GtkDialog-class.html).
* It's in the QtDialog API page [here](https://autokey.github.io/autokey/lib.scripting.QtDialog-class.html).

The problem exists in the **Usage** line in the current documentation:
* It's in the GtkDialog API page [here](https://autokey.github.io/api/gtkdialog.html).
* It's in the QtDialog API page [here](https://autokey.github.io/api/qtdialog.html).

To see the issue:
1. Examine the code for the [GTK method](https://github.com/autokey/autokey/blob/master/lib/autokey/scripting/dialog_gtk.py#L225) and the [Qt method](https://github.com/autokey/autokey/blob/master/lib/autokey/scripting/dialog_qt.py#L237) that should be used in the API calls.
2. Copy the **Usage** line from each of those links I just listed above into a text file.
3. Copy this correct **Usage** line that I recommended above into the same text file:
   ```
   Usage: dialog.calendar(title="Choose a date", format="%Y-%m-%d", date="YYYY-MM-DD", **kwargs)
   ```
4. Compare all five of those lines against each other.
5. Note that the four incorrect **Usage** lines from the above links call the nonexistent **calendar_dialog** method.
6. Note that the correct **Usage** line calls the **calendar** method.

You'll need to fix all of them before this can be closed.
